src/bundle/Resources/views/themes/admin/account/change_password/index.html.twig

```diff
@@ -1,42 +1,41 @@
-{% extends '@ibexadesign/ui/base.html.twig' %}
+{% extends '@ibexadesign/ui/edit_base.html.twig' %}
 
 {% form_theme form_change_user_password '@ibexadesign/account/change_password/form_fields.html.twig'  %}
 
 {% trans_default_domain 'user_change_password' %}
 
-{% block breadcrumbs_admin %}
-    {% include '@ibexadesign/ui/breadcrumbs.html.twig' with { 'items': [
-    { value: 'breadcrumb.user_preferences'|trans(domain='messages')|desc('User Preferences') },
-    { value: 'user_change_password.change_password'|trans|desc('Change Password') }
-    ]} %}
-{% endblock %}
+{% block main_container_class %}ibexa-main-container--edit-container ibexa-main-container--without-anchor-menu-items{% endblock %}
```

Comment:
Do we have to set these classes? They seem to be set by parent twig `'@ibexadesign/ui/edit_base.html.twig'`. Did I miss something? 
(here and in other places)